### PR TITLE
Rotation Count of Array -Python

### PR DESCRIPTION
**Pull request by U-c0de**
#1743  ISSUE

DO Check pull request

@Rohan27s @rish-singhal @yasharth291

### CODE REVIEW

**Comment by U-c0de** [comment]
DO Check pull request

@Rohan27s @rish-singhal @yasharth291

File: Code/Python/RotationCount.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+'''
+Test Case 1:
+            Input : lst = [4, 9, 11, 12, 5]
+            Output : 4
+
+Test Case 1:
+            Input : lst = [7, 9, 11, 12, 15]
+            Output : 0
+
+Test Case 1:
```
Comment (Rohan27s) [inline Code/Python/RotationCount.py:57]: correct the test number

File: Code/Python/RotationCount.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            Input : lst = [15, 18, 2, 3, 6, 12]
+            Output : 2
+
+Time Complexity: O(log n)
```
Comment (Rohan27s) [inline Code/Python/RotationCount.py:61]: mention that what is n here

**Review by Rohan27s — CHANGES_REQUESTED** [review]

**Comment by U-c0de** [comment]
Now check @Rohan27s @rish-singhal

**Comment by U-c0de** [comment]
Now check @Rohan27s @rish-singhal

**Review by Rohan27s — APPROVED** [review]

File: Algorithm/Number_theory/readme.md [hunk 3]
```diff
@@ -65,3 +65,4 @@
 - To find all prime number from 2 to N ----> [C++](Code/C++/Sieves_prime.cpp)
 - Geek-onacci Number----> [C++](/Code/C++/Geek-onacciNumber.cpp)
 - All Unique Prime Factors of any number N ----> [C++](/Code/C++/Unique_prime_factor.cpp)
```
Comment (Rohan27s) [inline Algorithm/Number_theory/readme.md:67]: Make changes in the readme file of Array

**Review by Rohan27s — CHANGES_REQUESTED** [review]

**Comment by U-c0de** [comment]
Now check @Rohan27s @rish-singhal

File: Algorithm/Number_theory/readme.md [hunk 4]
```diff
@@ -65,3 +65,4 @@
...
+- Rotation Count in a sorted array ----> [Python](/Code/Python/RotationCount.py)
```
Comment (Rohan27s) [inline Algorithm/Number_theory/readme.md:68]: remove this

**Review by Rohan27s — CHANGES_REQUESTED** [review]

**Comment by U-c0de** [comment]
Now check @Rohan27s @rish-singhal

**Review by Rohan27s — APPROVED** [review]

**Pull request merged**